youtube_dl/utils.py

```diff
@@ -3523,7 +3523,7 @@ def str_to_int(int_str):
     if isinstance(int_str, compat_integer_types):
         return int_str
     elif isinstance(int_str, compat_str):
-        int_str = re.sub(r'[,\.\+]', '', int_str)
+        int_str = re.sub(r'[,\.\s\+]', '', int_str)
```

Comment:
This fixes view count parsing for international sites that use a space separator (e.g. fr.pornhub.com) as seen here:

<img width="149" alt="Screen Shot 2020-02-16 at 4 22 25 PM" src="https://user-images.githubusercontent.com/107117/74615800-a636f900-50d8-11ea-9e6d-3c93a4699687.png">

This change can easily be reverted if it causes problems.